### DISCUSSION TITLE
Set Delay On Carousel

### DISCUSSION THREAD

**Discussion by adw259** [mdbootstrap/TW-Elements#1856] [Support from community] [answered]
I was wondering if there was any general way that the components that are integrated with JS can be manipulated? I'm simply trying to set a delay on when the carousel first starts so that it'll be out of sync with an adjacent carousel.

**Comment by juujisai** [comment, marked as answer]
Each option that is currently available for carousels can be found here: https://tailwind-elements.com/docs/standard/components/carousel/

As of the delay, we do not have such an option. Maybe you can initialize the Carousel via JavaScript after some timeout (`setTimeout`) ?
